src/Application.vala

```diff
@@ -220,8 +211,8 @@ public class AppCenter.App : Gtk.Application {
         }
 
         if (active_window == null) {
-            // Force a Flatpak cache refresh when the window is created, so we get new apps
-            update_manager.update_cache.begin (true);
+            // TODO: Think about this, maybe only refresh cache?
```

Comment:
I think we should leave the reasoning from the old comment here to make sure we know going forward that the reason we do this refresh is to make sure we fetch new apps when AppCenter is opened